### ISSUE TITLE
Allow drag’n’drop of folders

### ISSUE THREAD

**Issue by timpulver** [quodlibet/quodlibet#1704]
Would be great if Quod Libet would accept folders to be dropped on the playlist pane. 
It could then recursively go through all subfolders and load the included music files.

**Comment by lazka** [comment]
Sounds good.

* A dialog confirmation might be nice to prevent adding lots of stuff if there unnoticed are subdirectories
* And/Or make the adding process abortable.

**Comment by timpulver** [comment]
Yes, on some other music player (forgot which one it was, VLC maybe!?) there was a warning if you dropped more than ~100 files at once.
